README.md

```diff
@@ -136,6 +136,23 @@ Run migrations – this step will create tables for `auth` and `authtoken` apps:
 
     $ ./manage.py migrate
 
+### JSON Web Token Authentication
+`djoser` works out of the box with JSON web token authentication. You simply
```

Comment:
I think it might be useful to tell what additional packages are needed for this to work, e.g. **djangorestframework-jwt.**